### PR TITLE
Updated light theme visited link colour to make text visible

### PR DESCRIPTION
**Pull request by m158261**
## References

This fixes https://github.com/jupyterlab/jupyterlab/issues/15403

## Code changes

The visited link CSS variable in light theme has been changed to make the text visible. 

## User-facing changes

Link after and before visit - Light Theme
new visited link

Link after and before visit - Dark Theme
new visited link dark v2

## Backwards-incompatible changes

None

### CODE REVIEW

**Comment by krassowski** [comment]
Thank you @m158261 can you add screenshots in dark mode as well?

**Comment by krassowski** [comment]
So this is what I see in dark/light theme. I am not convinced weather the light mode contrast is enough and whether the dark mode contrast is not too much. Top not visited, bottom visited:

| Light | Dark |
|--|--|
| Screenshot from 2023-11-18 14-22-03 | Screenshot from 2023-11-18 14-21-59 |

**Comment by m158261** [comment]
Hi @krassowski, I have changed the visited link colour in light and dark themes for better contrast. I have used colours similar to the Google Chrome light/dark themes. Please let me know what you think. 

| Light | Dark |
|--------|----------|
| new visited link | new visited link dark |

| Google Light | Google Dark |
|--------|----------|
| google visited links light |  google visited links |

File: packages/theme-dark-extension/style/variables.css [hunk 1]
```diff
@@ -167,7 +167,7 @@ all of MD as it is not optimized for dense, information rich UIs.
   --jp-content-font-color2: rgba(255, 255, 255, 0.7);
   --jp-content-font-color3: rgba(255, 255, 255, 0.5);
   --jp-content-link-color: var(--md-blue-300);
-  --jp-content-link-visited-color: rgba(255, 255, 255, 0.3);
+  --jp-content-link-visited-color: var(--md-purple-400);
```
Comment (krassowski) [inline packages/theme-dark-extension/style/variables.css:170]: This fails WCAG AA. What about:

```suggestion
  --jp-content-link-visited-color: var(--md-purple-300);
```

**Review by krassowski — COMMENTED** [review]
That's definitely the right direction, but I think the dark theme would have too little contrast (3.34:1) so I propose to change it (upping it to 4.52:1).

**Comment by m158261** [comment]
> That's definitely the right direction, but I think the dark theme would have too little contrast (3.34:1) so I propose to change it (upping it to 4.52:1).

Looks good to me. I've updated the screenshot in the description.

**Comment by m158261** [comment]
I've had a play about with different colours, we could use #d497de which passes AA and AAA with a contrast ratio of 7.11. What do you think? 

d497de

**Review by krassowski — APPROVED** [review]
LGTM, thanks!

**Pull request merged**